### ISSUE TITLE
JS Version

### ISSUE THREAD

**Issue by pneb** [Its-Vichy/GoCycle#3]
https://gist.github.com/pneb/5d4c75b23de2205de8ae54c40ae62373

yeah

**Comment by Its-Vichy** [comment]
👋 Hi, thanks for your contribution. 
Can you please remove the insults from the script and replace them with something else ?

**Comment by pneb** [comment]
Yeah sure. Not a thing. It's been updated. I'm deeply apologizing for what I inputted as an example for the usage of the function.

**Comment by Its-Vichy** [comment]
☠️ No problems

**Comment by pneb** [comment]
Lol. Anyways, the shortened link of your discord server is invalid. Mind sending the direct discord link on my mail or here? contact@bern.codes

**Comment by Its-Vichy** [comment]
I have updated links into my readme (https://github.com/Its-Vichy/Its-Vichy)
